editor/src/components/canvas/ui-jsx-canvas.spec.tsx

```diff
@@ -1133,9 +1133,15 @@ export var ${BakedInStoryboardVariableName} = (props) => {
             width: 400px;
             height: 400px;
           \\"
+          data-paths=\\":utopia-storyboard-uid\\"
           data-uid=\\"utopia-storyboard-uid\\"
         >
-          <div data-uid=\\"ccc aaa scene-aaa\\">hello</div>
+          <div
+            data-uid=\\"ccc aaa scene-aaa\\"
+            data-paths=\\"utopia-storyboard-uid/scene-aaa:aaa :utopia-storyboard-uid/scene-aaa\\"
```

Comment:
This is missing a path, but I think that's right since the `Thing` isn't focused?

Reply:
this was fine, ccc is coming from an ARBITRARY_JS block